internal/storetesting/charm-repo/quantal/juju-gui/metadata.yaml

```diff
@@ -0,0 +1,47 @@
+# This file is part of the Juju GUI, which lets users view and manage Juju
+# environments within a graphical interface (https://launchpad.net/juju-gui).
+# Copyright (C) 2012-2013 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License version 3, as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+name: juju-gui
+summary: User interface for Juju (development)
+maintainer: Juju GUI Team <juju-gui@lists.launchpad.net>
+description: Web GUI for Juju
+tags: ["ops"]
+series:
+ - lucid
```

Comment:
this means lucid will be the default one in charm testing, please drop lucid out of the list.